changelogs/fragments/489_vmware_vm_inventory_port_cast_integer.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- vmware_vm_inventory - ensure self.port is integer (https://github.com/ansible-collections/community.vmware/pull/489).
```

```suggestion
- vmware_vm_inventory - ensure self.port is integer (https://github.com/ansible-collections/community.vmware/issues/488).
```
> Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.
[Creating a changelog fragment](https://docs.ansible.com/ansible/2.10/community/development_process.html#creating-a-changelog-fragment)

Since there is a corresponding issue, I think you should link to it instead of to the PR. Apart from that, the PR looks good to me.

---

Thanks @mariolenz. Next time I'll know what to do.